### ISSUE TITLE
It doesnt work anymore

### ISSUE THREAD

**Issue by Noobimation** [LostDragonist/steam-library-setup-tool#10]
I executed everything like the usage guide told me and it doesnt work.

**Comment by LostDragonist** [comment]
Seems to work for me.  The program opens.  I can add a new library.  The library shows up when I'm installing a game.  I even reinstalled Steam (note Steam will probably delete your games if you do this) to make sure it works with fresh files. /shrug

**Comment by fyndor** [comment]
I can confirm that it is still working.  I just tried it for the first time with an existing steam library.  Added a second folder in the C drive.  Make sure you create the folder first.  Not sure if it matters but I personally did.  And also make sure you right clicked and run as administrator.  Not sure exactly what he is doing but steam itself is almost certainly stored in program files so if you make modifications without admin permissions then it will create this "virtual" user level copy of the settings file this tool is modifying (making assumptions) and likely will not play well.  This is the reason why I want to make another library folder personally.  I don't want my games in a UAC controlled folder.  I installed a game with it (Morrowind!) and it worked fine.  You see two C drives because it isnt designed to have two folder for one drive so there is no other label other than C:.  I just have to know to use the 2nd one, but that is fine.  Thanks for making this tool <3

**Comment by MindFullOfLettuce** [comment]
Second library appeared after restarting steam

**Comment by techbot2342** [comment]
Worked for me. Legend @LostDragonist

**Comment by picarus1021** [comment]
This does work! The instructions are a bit outdated, because of updates with steam. You will follow the directions up until you add new row. It is likely that your top, greyed out, row is C:\Program Files (x86)\Steam. Your new row should be C:\Steam Library. I also added this new folder to C: before I used the program. To see if it worked, open steam and go to Steam>Settings>Storage (opposed to downloads), and you should have your main drive with your games that has a star on it. If you click on this, you can see a second drive, likely called the same thing but it is not the primary drive. When you go to reinstall your games, F3 and FNV for me, install them into this new drive.

**Comment by f6n73j** [comment]
Sadly, this just doesn't work for me. It just says "Error creating directories". I've tried the C: drive (folder created before) F: drive (same) and it just gives he same error. Admin, steam closed, .net dependencies installed, virus protection off. Any ideas how I could do this manually?

**Comment by picarus1021** [comment]
Unfortunately I don’t. Don’t forget to also run as administrator. I put the
new folder titled “Steam Library” directly in my C: drive. In the same area
where program86 is found for me. Then I used the tool to add a row and
browse to click on the new folder

**Comment by Red-De-vil** [comment]
> I executed everything like the usage guide told me and it doesn't work.

Maybe you didn't "Run it as Administrator", it worked for me with this. Date: 17th Nov 2024

**Comment by web-dev-3000** [comment]
> > I executed everything like the usage guide told me and it doesn't work.
> 
> Maybe you didn't "Run it as Administrator", it worked for me with this. Date: 17th Nov 2024

Yep as of June 2026 it is still working, if you had to run it as admin as you should because you should have your UAC high and be on a standard user account(you know or just use Linux), you will need to run steam as admin too else you shall get a permission error when downloading a game to your newly created steam folder.
